Formula/infracost.rb

```diff
@@ -0,0 +1,78 @@
+class Infracost < Formula
+  desc "Cost estimates for Terraform"
+  homepage "https://docs.infracost.io/"
+  url "https://github.com/infracost/infracost/archive/v0.4.1.tar.gz"
```

Comment:
Can we possibly use `v0.5.1` as that's the latest version?

Reply:
Sure, definitely.